Description: Disable a convoluted "build log buffering" mechanism which was causing problems.
Author: Ximin Luo <infinity0@debian.org>
Forwarded: not-needed
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/sage/src/sage_setup/docbuild/sphinxbuild.py
+++ b/sage/src/sage_setup/docbuild/sphinxbuild.py
@@ -205,8 +205,9 @@
     saved_stderr = sys.stderr
 
     try:
-        sys.stdout = SageSphinxLogger(sys.stdout, os.path.basename(output_dir))
-        sys.stderr = SageSphinxLogger(sys.stderr, os.path.basename(output_dir))
+        # Sage is too smart for their own good, this poorly-implemented "file" replacement breaks stuff
+        #sys.stdout = SageSphinxLogger(sys.stdout, os.path.basename(output_dir))
+        #sys.stderr = SageSphinxLogger(sys.stderr, os.path.basename(output_dir))
         sphinx.cmdline.main(sys.argv)
     finally:
         sys.stdout = saved_stdout
